src/oci.h

```diff
@@ -143,7 +143,7 @@ enum oci_status {
 	OCI_STATUS_RUNNING,
 	OCI_STATUS_PAUSED,
 	OCI_STATUS_STOPPED,
-	OCI_STATUS_STOPPING,
+	OCI_STATUS_SIGNALING,
```

Comment:
Again is this value part of OCI spec? Might be useful to have this in its own commit with explanation for introducing this value.